app/components/NavBar/Settings/PlayerList.tsx

```diff
@@ -1,157 +1,73 @@
 'use client';
+
+import {
+    acceptPlayer,
+    denyPlayer,
+    kickPlayer,
+} from '@/app/hooks/server_actions';
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import PendingPlayers from './PendingPlayers';
+import { fetchPendingPlayers } from '@/app/utils/fetchPlayers';
```

Comment:
Should just call the same function from the server_actions and we shouldnt have the fetchPlayers file in utils